brainrender/atlas.py

```diff
@@ -0,0 +1,135 @@
+from bg_atlasapi.bg_atlas import BrainGlobeAtlas
+from vedo import Plane
+import numpy as np
+
+from brainrender import settings
+from .actor import Actor
+from ._io import load_mesh_from_file
+from ._utils import return_list_smart
+
+
+class Atlas(BrainGlobeAtlas):
+    def __init__(self, atlas_name=None):
+        """
+            Brainrender's Atlas class subclasses BrainGlobeAtlas
+            to add methods to get regions meshes as Actors
+            and to get a plane at a given point and normal.
+
+            :param atlas_name: str, atlas name from brainglobe's atlas API atlases
+        """
+        atlas_name = atlas_name or settings.DEFAULT_ATLAS
+        BrainGlobeAtlas.__init__(
+            self, atlas_name=atlas_name, print_authors=False
+        )
+
+    def get(self, _type, *args, **kwargs):
```

Comment:
this getter looks a bit funny to me, what's its purpose? Sounds a bit too general to be useful, I guess that stuff will always be like get_plane or get_region

Reply:
yep, I first put it in thinking that more stuff could be `got` from the atlas but it is now redundant, removing.